rfc/svs-vamana.md

```
@@ -0,0 +1,641 @@
+| Milvus | DiskANN (Vamana family) | Scalar/Product quantization | Limited |
+| Qdrant | No (HNSW only) | Scalar/Product quantization | No |
+| Weaviate | No (HNSW only) | Product quantization | No |
+| **valkey-search + SVS** | **Yes (SVS_VAMANA)** | **LVQ + LeanVec** | **x86_64 AVX-512/AVX2** |
+
+## Specification
+
+### FT.CREATE with ALGORITHM SVS
+
+The `SVS_VAMANA` algorithm is selected via the `ALGORITHM` parameter in the `VECTOR` field specification of `FT.CREATE`:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)